src/services/api.ts

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const api = axios.create({
```

Comment:
no, I have shown on the call and its in the doc we use rtk -  https://redux-toolkit.js.org/tutorials/rtk-query